src/referencePackages/Directory.Build.props

```diff
@@ -1,6 +1,11 @@
 <Project>
 
   <Import Project="..\..\Directory.Build.props" />
+  
+  <ItemGroup>
+    <!-- Attributes for all Reference Packages -->
+    <AssemblyMetadata Include="source" Value="source-build-reference-packages" />
```

Comment:
I don't think that's the correct usage. Please double check that works (open a compiled assembly with ilspy / ildasm).

Reply:
Here's an example: https://github.com/dotnet/runtime/blob/53830e2608467ed6c2be38a782cc07803d960f53/eng/versioning.targets#L12-L15

Reply:
https://learn.microsoft.com/en-us/visualstudio/msbuild/common-msbuild-project-items?view=vs-2022#assemblymetadata

https://github.com/dotnet/docs/blob/main/docs/core/project-sdk/msbuild-props.md#assemblymetadata

These documents suggest that this usage is correct.

Reply:
Interesting, I didn't know that.